stories/1-Icon.stories.tsx

```diff
@@ -0,0 +1,331 @@
+export const Plus = <PlusIcon />;
+export const ProjectEvent = () => <ProjectEventIcon />;
+export const ProjectLink = () => (
+  <>
+    <ProjectLinkIcon type="issue" />
+    <ProjectLinkIcon type="homepage" />
+    <ProjectLinkIcon type="ci" />
+    <ProjectLinkIcon type="scm" />
+    <ProjectLinkIcon type="something else" />
+  </>
+);
+export const PullRequest = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <PullRequestIcon />
+  </ThemeProvider>
+);
+export const Qualifier = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <QualifierIcon qualifier="app" />
+    <QualifierIcon qualifier="brc" />
+    <QualifierIcon qualifier="dev" />
+    <QualifierIcon qualifier="dir" />
+    <QualifierIcon qualifier="fil" />
+    <QualifierIcon qualifier="svw" />
+    <QualifierIcon qualifier="trk" />
+    <QualifierIcon qualifier="uts" />
+    <QualifierIcon qualifier="vw" />
+
+    <p>Deprecated</p>
+    <QualifierIcon qualifier="cla" />
+    <QualifierIcon qualifier="dev_prj" />
+    <QualifierIcon qualifier="lib" />
+    <QualifierIcon qualifier="pac" />
+
+    <p>Other</p>
+    <QualifierIcon qualifier={null} />
+    <QualifierIcon qualifier={undefined} />
+  </ThemeProvider>
+);
+export const Recommended = () => <RecommendedIcon />;
+export const RuleScope = () => <RuleScopeIcon />;
+export const SCM = () => <SCMIcon />;
+export const Search = () => <SearchIcon />;
+export const SecurityHotspot = () => <SecurityHotspotIcon />;
+export const Severity = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <SeverityIcon severity="blocker" />
+    <SeverityIcon severity="critical" />
+    <SeverityIcon severity="major" />
+    <SeverityIcon severity="minor" />
+    <SeverityIcon severity="info" />
+  </ThemeProvider>
+);
+export const ShortLivingBranch = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <ShortLivingBranchIcon />
+  </ThemeProvider>
+);
+export const SortAsc = () => <SortAscIcon />;
+export const SortDesc = () => <SortDescIcon />;
+export const Status = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <StatusIcon status="open" />
+    <StatusIcon status="confirmed" />
+    <StatusIcon status="reopened" />
+    <StatusIcon status="resolved" />
+    <StatusIcon status="closed" />
+    <StatusIcon status="to_review" />
+    <StatusIcon status="in_review" />
+    <StatusIcon status="reviewed" />
+  </ThemeProvider>
+);
+export const Tags = () => <TagsIcon />;
+export const TestStatus = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <TestStatusIcon status="ok" />
+    <TestStatusIcon status="failure" />
+    <TestStatusIcon status="error" />
+    <TestStatusIcon status="skipped" />
+  </ThemeProvider>
+);
+export const Tree = () => <TreeIcon />;
+export const Treemap = () => <TreemapIcon />;
+export const Visible = () => <VisibleIcon />;
+export const Vulnerability = () => <VulnerabilityIcon />;
+export const Warning = () => (
+  <ThemeProvider theme={mockedTheme}>
+    <WarningIcon />
+  </ThemeProvider>
+);
```

Comment:
I have to admit I'm a bit put-off by this story system, in which you have to import and export every single component.
Isn't there an easier way, and wrap all of them in the theme provider?

Reply:
That's something we can discuss at the web-guild goûter !

Reply:
I had the same feeling, arf, adding every component manually is such a pain.

Reply:
Waw that's cool! But indeed it seems to be a bit of a pain to have to add them all manually, do you think it would be possible to mix this with tests ?
Or generate this file ?